finetune/base_models/gpt2/featurizer.py

```diff
@@ -204,9 +204,10 @@ def gpt2_featurizer(
         pool_idx = tf.cast(
             tf.argmax(tf.cast(tf.equal(X[:, :, 0], clf_token), tf.float32), 1), tf.int32
         )
+        batch, length, _ = shape_list(X)
         clf_h = tf.gather(
             clf_h,
-            tf.range(shape_list(X)[0], dtype=tf.int32) * config.max_length + pool_idx,
+            tf.range(batch, dtype=tf.int32) * length + pool_idx,
```

Comment:
Oh yikes -- was this straight up broken previously? Or were we updating config.max_length all the time or something.